pkg/core/transaction/witness_scope.go

```diff
@@ -53,3 +54,41 @@ func ScopesFromString(s string) (WitnessScope, error) {
 	}
 	return result, nil
 }
+
+// scopesToString converts witness scope to it's string representation. It uses
+// `, ` to separate scope names.
+func scopesToString(scopes WitnessScope) string {
+	if scopes == Global {
+		return "Global"
+	}
+	var res []string
```

Comment:
Maybe it's not worth creating a slice here. Not that this code is in any hot-path, though.